<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Uploader</class>
 <widget class="QMainWindow" name="Uploader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Uploader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="5" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Wyślij</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="7">
     <widget class="QPlainTextEdit" name="linksListEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Odstęp czasowy</string>
      </property>
      <property name="buddy">
       <cstring>spinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Losowy czas</string>
      </property>
      <property name="buddy">
       <cstring>randomSpin</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QSpinBox" name="randomSpin">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>414</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Dodaj...</string>
    </property>
    <addaction name="actionAddFile"/>
    <addaction name="actionAddTemplate"/>
   </widget>
   <widget class="QMenu" name="menuEdytuj">
    <property name="title">
     <string>Edytuj...</string>
    </property>
    <addaction name="actionEditFile"/>
    <addaction name="actionEditTemplate"/>
    <addaction name="actionEditMirror"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuAdd"/>
   <addaction name="menuEdytuj"/>
   <addaction name="menuPomoc"/>
  </widget>
  <action name="actionAddFile">
   <property name="text">
    <string>Dodaj plik</string>
   </property>
  </action>
  <action name="actionAddTemplate">
   <property name="text">
    <string>Dodaj szablon</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
  <action name="actionEditFile">
   <property name="text">
    <string>Edytuj pliki</string>
   </property>
  </action>
  <action name="actionEditTemplate">
   <property name="text">
    <string>Edytuj szablony</string>
   </property>
  </action>
  <action name="actionEditMirror">
   <property name="text">
    <string>Edytuj mirrory</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Ustawienia</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spinBox</tabstop>
  <tabstop>randomSpin</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>linksListEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
